src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/controllers/SupplementaryDetailsController.java

```diff
@@ -32,7 +34,7 @@ public SupplementaryDetailsController(SupplementaryDetailsService supplementaryD
         response = String.class,
         authorizations =
             {
-                @Authorization(value = "ServiceAuthorization")
+                @Authorization(value = "serviceAuthorization")
```

Comment:
Capital letter is ok

Reply:
I put back the header with the capital letter